src/backend/distributed/commands/owned.c

```diff
@@ -88,3 +88,36 @@ PreprocessDropOwnedStmt(Node *node, const char *queryString,
 
 	return NodeDDLTaskList(NON_COORDINATOR_NODES, commands);
 }
+
+
+List *
+PreprocessReassignOwnedStmt(Node *node, const char *queryString,
```

Comment:
need to move this down to Postprocess phase

Reply:
Fixed